marimbabot_planning/launch/marimbabot.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+    <node name="lilypond_to_hitpoint" pkg="marimbabot_planning" type="lilypond_to_hitpoint.cpp"/>
```

Comment:
I don't know if this should be in its own node in the end or if it should be included as a library in the motion.